lib/V8/v8-utils.cpp

```diff
@@ -1907,13 +1890,26 @@ static void JS_Log(v8::FunctionCallbackInfo<v8::Value> const& args) {
         logLineVec.push_back(*message);
       }
     }
+
     for (auto& message : logLineVec) {
-      if (topic == nullptr) {
-        LOG_TOPIC_RAW(ll, Logger::FIXME) << prefix << message;
+      if (ls == "fatal") {
+        prefix = "FATAL! ";
+        LOG_TOPIC(ERR, topic) << prefix << message;
```

Comment:
can't we have a unordered map to look up this mapping, and have this code here just once? /me hates Rumpelstilskin lookups.

Reply:
There is only one macro left as @jsteemann requested. This macro requires `FATAL`, `ERR`, `WARN`, `DEBUG` or `TRACE`. In case we decide to introduce unique ids for JavaScript we need to change something here.